### PR TITLE
Add utilization trend card pattern

### PR DESCRIPTION
**Pull request by serenamarie125**
Add the pattern as it currently exists on patternfly.org

### CODE REVIEW

**Comment by andresgalante** [comment]
Hi Serena, this PR also needs update on the relative paths of the links.

**Comment by serenamarie125** [comment]
@LHinson PR has been updated.  LMK if anything else needs to be addressed!

File: pattern-library/cards/utilization-trend-card/design/overview.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Utilization Trend Card
+
+## Overview
+Utilization cards display the current utilization and a trend of a single metric. The most common use case for this pattern is seen in a dashboard.
+
+Jump to [Card with a Single Metric](#pattern-example) or [Card with Multiple Metrics](#pattern-variation-example)
+
+## Examples
+
+### Pattern Example
```
Comment (LHinson) [inline pattern-library/cards/utilization-trend-card/design/overview.md]: Should say "Card with a Single Metric" not "Pattern Example"

File: pattern-library/cards/utilization-trend-card/design/overview.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+![Pattern Example](img/utilization-trend-card-example-1.png)
+
+### Pattern Variation Example
```
Comment (LHinson) [inline pattern-library/cards/utilization-trend-card/design/overview.md]: Should say "Card with Multiple Metrics"

**Comment by serenamarie125** [comment]
Updated to put in the correct example titles @LHinson

**Comment by LHinson** [comment]
Great. Thanks @serenamarie125

**Pull request merged**